Update templates/LABELS.md

0xFelix opened the pull request:
<!--  Thanks for sending a pull request!  Here are some tips for you:
1. Follow the instructions for writing a release note from k8s: https://git.k8s.io/community/contributors/guide/release-notes.md
-->

**What this PR does / why we need it**:

This file is an additional maintenance burden with no real use.

**Release note**:
<!--  Write your release note:
1. Enter your extended release note in the below block. If the PR requires additional action from users switching to the new release, include the string "action required".
2. If no release note is required, just write "NONE".
-->
```release-note
None
```

0xFelix commented:
/cc yaacov

ksimon1 commented on templates/LABELS.md:
```diff
@@ -1,97 +0,0 @@
-Some of them are not going to be used immediately and serve
-as examples.
-
-## Deprecated templates
```
delete only this paragraph, not whole file

ksimon1 commented:
/ok-to-test

yaacov commented:
> @yaacov Could you have a look at this? Is this file still needed by UI?

It's just the documentation right ?

if it's not just documentation:
the UI use `deprecated` label, to not show it to users
the UI use the `os.template.kubevirt.io`, to guess the os name
workload and flavor are used to desplay nice information about the template

we need this labels in 4.10
we can deprecate them in 4.11, are the getting deprecated ?

0xFelix commented:
@yaacov Yes, it is only documentation. Sorry for the misunderstanding. It is tedious to update the versions in `LABELS.md` manually. The labels are definitely going to stay.

I updated the PR so LABELS.md is kept and changed the tedious parts instead.

0xFelix commented:
/retest

akrejcir commented:
/approve
